test/test_strategies.rb

```diff
@@ -189,6 +189,7 @@ def test_xml
       "#{samples_path}/XML/some-ideas.mm",
       "#{samples_path}/XML/GMOculus.project.gmx",
       "#{samples_path}/XML/obj_control.object.gmx",
+      "#{samples_path}/XML/MainView.axaml"
```

Comment:
Fixed the failing test here. We need this because your file doesn't start with `<?xml...` so we need to tell the XML strategy tests to ignore it.